docs/en/02_Features/active_directory_single_sign_on.md

```diff
@@ -11,186 +11,7 @@ further prompts when they switch applications.
-    Config::inst()->update('SAMLConfiguration', 'SP', [
-        'entityId' => 'https://stackname.cwp.govt.nz/',
-        'privateKey' => '../../certs/saml.pem',
-        'x509cert' => '../../certs/saml.crt'
-    ]);
-}
-```
-	
-### Disable the basic auth for an intranet site
-
-CWP by default prevents access to the UAT environment by
-restricting access people with logins. This prevents the ADFS
-server from [accessing the metadata](https://github.com/silverstripe/silverstripe-activedirectory/blob/master/docs/en/adfs.md#select-data-source) on this environment.
-
-Since the intranet site should only be reachable over a VPN link,
-this feature can be disabled.
-
-```yaml
----
-Name: mysitesecuritytest
-After: '#cwpsecuritytest'
----
-# The basic authentication popup must be disabled in the UAT
-# environment, otherwise the ADFS server can't get the
-# metadata of the SAML Service Provider
-SilverStripe\Security\BasicAuth:
-  entire_site_protected: false
-```
-
-## Frequently Asked Questions
-
-### Why not use Kerberos?
-
-Generally a Domain Controller (DC) uses Integrated Windows
-Authentication (IWA, usually Kerberos or NTLM) to authenticate a
-user to the application they are trying to access.
-
-However this only works if:
-
- - The user has direct access to the Domain Controller (DC)
- - The users device supports IWA
-
-For security reasons the DC should never be exposed to the public
-internet so that would prevent users from using a public and
-private resource and had to login to them individually.
-
-The ADFS will step in as a web endpoint on a secure HTTPS
-connection that uses the DC as a Identity Provider.
-
-Mobile devices often can't provide IWA details so that would
-prevent Single sign-on for mobile devices.
-
-Since ADFS is exposed as a website it can automatically login users
-if their device support IWA or can fall back to a standard login
-form.
-
-Allowing additional services to use the ADFS as an authentication
-endpoint doesn't require opening network ports or ip whitelisting.
-
-Other reasons for not using Kerberos also include:
-
- - Open source web servers like Apache have limited support for Kerberos
-   through extensions, but are not officially supported and can
-   be difficult to setup and configure.
- - Clocks between AD and the web server have to be perfectly matched
-   in order for it to work without failures.
-
-### Why does the intranet site need access to the AD server?
-
-It's strictly not necessary for authentication, but to be able to
-synchronise a custom Staff Directory Page with the Active
-Directory, the SilverStripe site needs to synchronise daily without
-any user intervention. This can only be done if the SilverStripe
-site can connect to the AD.
-
-## See Also
-
- * [Authentication via RealMe](realme_authentication)
+You might be able to adapt the contents of the Silverstripe CMS 3 guide  to work with either the
```

Comment:
This doesn't really make sense, where is the guide? Wouldn't it make sense to just say "Use one of these modules instead" or even just remove this page?

Reply:
Alright, I reinstated the guide, but made the note more prominent.